src/main/java/lotto/domain/LottoRank.java

```diff
@@ -0,0 +1,45 @@
+package lotto.domain;
+
+import java.math.BigDecimal;
+import java.util.Arrays;
+import lotto.application.service.vo.MatchingInfo;
+
+public enum LottoRank {
+    FIFTH(3, false, BigDecimal.valueOf(5000)),
+    FOURTH(4, false, BigDecimal.valueOf(50000)),
+    THIRD(5, false, BigDecimal.valueOf(1500000)),
+    SECOND(5, true, BigDecimal.valueOf(30000000)),
+    FIRST(6, false, BigDecimal.valueOf(2000000000));
```

Comment:
자바에서 숫자를 사용할때 구분자로 "_ "를 사용할 수 있습니다 이를 통해 더 직관적으로 숫자를 확인할 수 있습니다. 이는 코드 가독성을 위한 표시일 뿐 실제 값에는 영향을 주지 않습니다.


```suggestion
FIRST(6, false, BigDecimal.valueOf(2_000_000_000));
```

Reply:
아하 그렇군요.. 처음 알았습니다!! 감사합니다!